data/topics/1/topic.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 title: "UI / Web"
 description: "Verstehen und Entwickeln von Webseiten"
-year calendar-weeks:
+year-calendar-weeks:
```

Comment:
year_calendar_weeks: ...
snake_case :smile: 

I know ... Jonas added it like this ...